include/pmacc/cuplaHelper/ValidateCall.hpp

```diff
@@ -31,27 +31,27 @@ namespace pmacc
 {
 
 /**
- * Print a cuda error message including file/line info to stderr
+ * Print a cupla error message including file/line info to stderr
  */
 #define PMACC_PRINT_CUDA_ERROR(msg) \
     std::cerr << "[CUDA] Error: <" << __FILE__ << ">:" << __LINE__ << " " << msg << std::endl
 
 /**
- * Print a cuda error message including file/line info to stderr and raises an exception
+ * Print a cupla error message including file/line info to stderr and raises an exception
  */
-#define PMACC_PRINT_CUDA_ERROR_AND_THROW(cudaError, msg) \
+#define PMACC_PRINT_CUDA_ERROR_AND_THROW(cuplaError, msg) \
     PMACC_PRINT_CUDA_ERROR(msg);                         \
-    throw std::runtime_error(std::string("[CUDA] Error: ") + std::string(cudaGetErrorString(cudaError)))
+    throw std::runtime_error(std::string("[CUDA] Error: ") + std::string(cuplaGetErrorString(cuplaError)))
```

Comment:
Perhaps rename this and the following macro with CUDA as well? Or at least renaming in the output `[CUDA] Error` to `[Cupla] Error` makes sense to me.